### PR TITLE
DESK-1090 DESK-1068 DESK-1100 DESK-1098 license tests and updates

### PR DESCRIPTION
**Pull request by JamieRuderman**
### Description

- Fix connections sorting
- Fix binary version errors
- Support r3 custom icon
- Removing un-needed disable underline commands
- Settings test page addition
- Update display of licensing violations
- Add license chip display to service details
- Test UI allow hiding of highlights
- Enhance inline settings components
- Improve styling of inline reset
- License display improvements with copy license key
- Two panel layout for settings page
- Fixing settings page routing
- Adding connection count to side bar
- Clearing search state on logout
- fixing double panel width setting on changing pages
- Update mac system try icon size to match new os icon sizes
- Connection clear logic protect from offline and empty states
- Logs pages refactor
- Showing license notices for remote.it plans
- Adding badge to licensing menu item
- Hiding connections in remote ui.
- Fixing initial duration display end time calculation
- Fixing display of no AWS subscription and copy key button
- Fix updating connection bug
- Fixing cli update detection for windows
- Clarify device setup message in settings page

### Ticket

<https://remoteit.atlassian.net/browse/DESK-1090>
<https://remoteit.atlassian.net/browse/DESK-1068>
<https://remoteit.atlassian.net/browse/DESK-1100>
<https://remoteit.atlassian.net/browse/DESK-1098>

### CODE REVIEW

File: frontend/src/components/DoublePanel.tsx [hunk 1]
```diff
@@ -57,8 +57,17 @@ export const DoublePanel: React.FC<Props> = ({ primary, secondary, resize }) =>
     event.preventDefault()
     window.removeEventListener('mousemove', onMove)
     window.removeEventListener('mouseup', onUp)
-    ui.set({ [`${resize}PanelWidth`]: primaryRef.current?.offsetWidth || width })
+    console.log('SET PANEL WIDTH', resize, panelWidth, primaryRef.current?.offsetWidth || width)
```
Comment (mauricio0000) [inline frontend/src/components/DoublePanel.tsx:60]: dev log

File: frontend/src/components/Formats.tsx [hunk 2]
```diff
@@ -15,6 +16,15 @@ export const Formats = {
   round: (value: number) => {
     return Math.round(value * 10) / 10
   },
+  chip: (value: ILicenseChip) => {
+    return (
+      <Chip
+        style={{ color: value.color, backgroundColor: value.background, fontFamily: 'Roboto Mono' }}
```
Comment (mauricio0000) [inline frontend/src/components/Formats.tsx:22]: could you use a className?

**Review by mauricio0000 — APPROVED** [review]

File: frontend/src/components/DoublePanel.tsx (hunk 1 above)
Comment (JamieRuderman) [reply frontend/src/components/DoublePanel.tsx:60]: Thanks!

File: frontend/src/components/Formats.tsx (hunk 2 above)
Comment (JamieRuderman) [reply frontend/src/components/Formats.tsx:22]: Yes probably could - would have to pass properties to it.

**Pull request merged**